GuardiansoftheSpectrum/app/src/main/res/layout/activity_myinfo.xml

```diff
@@ -7,6 +7,17 @@
                 android:layout_height="match_parent"
                 android:weightSum="1">
 
+    <!-- <include layout="@layout/toolbar" /> -->
```

Comment:
Also if you want, you can simply uncomment this line and not have to have the definition below. the toolbar.xml file contains this information already